deployment/k8s/charts/aggregates/migrationAssistantWithArgo/templates/resources/otel/simpleCollectorAsIndependentDaemonset.yaml

```diff
@@ -94,6 +94,9 @@ kind: ServiceMonitor
 metadata:
   name: otel-collector-monitor
   namespace: {{ .Release.Namespace }}
+  annotations:
```

Comment:
Can we have all of these collected in a list to know what order we've assigned different components with a comment that points to that list?

Reply:
I'm curious how we can know that "1" is a good place amount the 0, -1, -10, -4's we've got in our codebase